js/components/reuse/card.vue

```diff
@@ -64,7 +64,18 @@ import placeholders from 'helpers/placeholders';
 import moment from 'moment';
 import config from 'config';
 
+const MASK = [
```

Comment:
Arrays, joined when necessary, not during the definition otherwise we'll have to split if again if we need to merge/modify it later.